db.json

```diff
@@ -3552,6 +3552,20 @@
         }, 
         "nid": 119
     }, 
+    "SceNpDrmPackage": {
+        "modules": {
+            "SceNpDrmPackage": {
+                "functions": {
+                    "_sceNpDrmPackageCheck": 2715321850,
+                    "_sceNpDrmPackageDecrypt": 3606076108
+                },
+                "kernel": false,
+                "nid": 2287029682,
+                "variables": {}
+            }
+        },
+        "nid": 120
```

Comment:
I think that this should be done with the massive overhault, including the _ change.